src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -3974,6 +3984,27 @@ type Trip implements Node {
     ): [Alert]
 }
 
+"""
+Trip on a specific date
+"""
+type DatedTrip implements Node {
```

Comment:
When we talked about this in the dev meeting, I was under the impression that @optionsome wanted to create a completely new type where the trip is "anchored" to a date rather than reusing the generic `Trip` whose fields change depending whether it is on a date or not.

Reply:
Done